heroku/cli
# refactor: migrate access:index to oclif/core

Migrates access/index to oclif/core.

This does change the table format a bit.
v5 version:
<img width="521" alt="image" src="https://github.com/heroku/cli/assets/12739849/66af10b6-1a7d-4454-8d60-fd2218ee7b29">

updated version
<img width="538" alt="image" src="https://github.com/heroku/cli/assets/12739849/8c4fec6a-d649-494a-b87d-ee26ff2ffcf8">

## Test
- check out branch and run `yarn` and `yarn build`
- run `./bin/run access -a APP-NAME`
    - The table will look different than the current version, but it should contain the same information
- run `./bin/run access -a APP-NAME --json`
    - The json output should look the same as the current version.
